pygfx/helpers/_gizmo.py

```diff
@@ -0,0 +1,233 @@
+"""
+A transform gizmo to manipulate world objects.
+"""
+
+
+import numpy as np
+import pygfx as gfx
+from ..objects import WorldObject
+from pygfx.linalg import Vector3
+
+
+class TransformGizmo(WorldObject):
+    """
+    A gizmo object that can be used to manipulate (i.e. transform) a world object.
+
+    Aguments:
+        object_to_control (WorldObject): the object to transform with the gizmo.
+        screen_size (float): the approximate size of the widget in logical pixels.
```

Comment:
is it the size of the diagonal?

Reply:
No, more like the average size (in x and y).

Reply:
So it's both the width and the height?